### PR TITLE
Feature get all entries

### PR DESCRIPTION
**Pull request by Kajochi**

### CODE REVIEW

File: backend/src/lombok.config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# tells Lombok that this is the root directory and that it shouldn’t search parent directories for more configuration files
+config.stopBubbling = true
+# tells Lombok to add @lombok.Generated annotation to all generated methods
```
Comment (Ben-21) [inline backend/src/lombok.config]: Please no code comments 😜

File: backend/src/main/java/de/neuefische/capstone/backend/entries/EntriesService.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package de.neuefische.capstone.backend.entries;
+
+import de.neuefische.capstone.backend.model.Entry;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+@Service
+public class EntriesService {
+
+    EntriesRepo entriesRepo;
```
Comment (Ben-21) [inline backend/src/main/java/de/neuefische/capstone/backend/entries/EntriesService.java]: Have you considered making this field final?

File: backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Data
+@RequiredArgsConstructor
```
Comment (Ben-21) [inline backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java]: @Data already includes @RequiredArgsConstructor ;-)

File: backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+    private String description;
+
+    private LocalDate date;
+
+    private BigDecimal amount;
+
+    private Category category;
+
+    private Interval interval;
```
Comment (Ben-21) [inline backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java]: This fields can also be final, as you are going to save them in a database and not going to change their values inside your java code, I suppose.

**Review by Ben-21 — APPROVED** [review]
Looks extremely good to me :)

File: backend/src/lombok.config (hunk 1 above)
Comment (Kajochi) [reply backend/src/lombok.config]: alright...:)

File: backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java (hunk 3 above)
Comment (Kajochi) [reply backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java]: i remember, we talked about it xD

File: backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java (hunk 4 above)
Comment (Kajochi) [reply backend/src/main/java/de/neuefische/capstone/backend/model/Entry.java]: this could be work

**Pull request merged**